webpack.config.js

```diff
@@ -48,14 +49,12 @@ var commonPlugins = [
 ];
 
 module.exports = {
-    // In development, bundle styles together with the code so they can also
-    // trigger hot reloads. In production, put them in a separate CSS file.
-
-    // @babel/polyfill is a polyfill. If you only need to support modern browsers, you can remove it.
+    // In development, have two different entries to speed up hot reloading.
+    // In production, have a single entry but use mini-css-extract-plugin to move the styles to a separate file.
     entry: isProduction ? {
-        app: ["@babel/polyfill", CONFIG.fsharpEntry, CONFIG.cssEntry]
```

Comment:
Is it included by the `"corejs": 3` changes?

Reply:
Yes, the babel loader complains now if I don't use the "corejs" option when using "useBuiltIns". And if I use "corejs" it says I must make sure the specific version is installed. Not sure what's the situation of @babel/polyfill now.